SBOM-Catalog/public/data.yaml

```diff
@@ -578,3 +578,19 @@
     - Validate
   Type:
   Language:
+
+- Name: GUAC
+  Link: https://github.com/guacsec/guac
+  Publisher: GUAC (OpenSSF)
+  License: OpenSource
+  Standards:
+    - CycloneDX
+    - SPDX
+  Abilities:
+    - Consume
```

Comment:
thanks for adding this @funnelfiasco , i wonder if "Compare" would fit here as well, since guac does provide some utility to compare SBOMs.

Reply:
Thanks! I opened #66 to add that since this PR was already merged